git_glossary.md

```diff
@@ -0,0 +1,31 @@
+# Git Glossary 
+
+This git glossary contains definitions of terms used in this repository.
+
+There is another dcoument that describes a mental model of git that
+when understood can help one master git concepts at a deeper level. 
+
+## commit
+As a noun: A single point in history of a repository. The entire history of a project is represented as a set
+of interrelated commits. Each commit is uniquely identified by a 40-character hexadecimal string which is created using
+a cryptographic hash function, also referred to has the SHA of the commit. Sometimes the SHA is abbreviated to the 
```

Comment:
Can we remove "cryptographic"? I think i understand what we are saying (that it's a hash function commonly used for cryptography) but am concerned that this might confuse users, maybe we can just say "hash function"? Kind Regards, Tord

Reply:
Done